maintenance/wikia/cronjobs/updateFandomShopCache.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Get data from intentX API and update memcache 
+ *
+ * @group cronjobs
+ * @see update-fandom-shop-cache.yaml
+ *
+ */
+
+require_once __DIR__ . '/../../Maintenance.php';
+
+class updateFandomShopCache extends Maintenance {
+
+    public function execute() {
+        global $wgFandomShopMap, $wgFandomShopMapDev;
+
+        echo 'Running maintenence script';
+
+        $logger = \Wikia\Logger\WikiaLogger::instance();
+        $logger->info( 'Updating Fandom Shop Cache' );
+
+        if ( Wikia::isDevEnv() ) {
```

Comment:
few comments here:

- you should override the $wgFandomShopMap value in dev.php, branching the code based on the env is discouraged
- judging by the https protocol, are you going through Fastly? which means a job executed on k8s will go though fastly servers border and back to k8s? That far from being efficient and also you risk the response will be cached on CDN so you won't get fresh data (unless you set the caching headers)
- actually why are you using curl instead of running this job in context of a selected wiki?

Reply:
Thanks Mech. We're going to skip the maintenance script for now and look for a better approach next week.